l10nzh.properties

```diff
@@ -2,13 +2,19 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=1.11,1.12,1.13,1.14,1.15,1.16,1.17,1.18
-publicVersions=1.19,1.20,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29
+publicVersions=1.19,1.20,1.21,1.22,1.23,1.24,1.25,1.26,1.27,1.28,1.29,8.0
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
+8.0.description=Support SonarQube 8.0
+8.0.sqVersions=[8.0,LATEST]
+8.0.date=2019-11-14
+8.0.changelogUrl=https://github.com/SonarQubeCommunity/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-8.0
+8.0.downloadUrl=https://github.com/SonarQubeCommunity/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-8.0/sonar-l10n-zh-plugin-8.0.jar
+
 1.29.description=Support SonarQube 7.9
-1.29.sqVersions=[7.9,LATEST]
+1.29.sqVersions=[7.9,7.9]
```

Comment:
You probably want to change the end of this range to 7.9.* so it works with 7.9.1, 7.9.2, ...